Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py

```diff
@@ -370,7 +374,7 @@ def setup__(self, step, stepName, stepDict, k, properties):
         elif 'ALCA' in step: stepDict[stepName][k] = None
         elif 'HARVEST' in step: stepDict[stepName][k] = merge([{'-s': 'HARVESTING:@trackingOnlyValidation+@pixelTrackingOnlyDQM'}, stepDict[step][k]])
     def condition_(self, fragment, stepList, key, hasHarvest):
-        return ('2017' in key or '2018' in key or '2021' in key or '2026' in key) and ('FS' not in key)
+        return ('2017' in key or '2018' in key or '2021' in key or '2026' in key or 'HI' in key) and ('FS' not in key)
```

Comment:
Just my curiosity, this includes almost all the keys except 2022, 2023, and 2024. Why? 

However, not a showstopper for this PR. It can enable later.

Reply:
I think it's because this has grown one piece at a time...

I guess by now we could drop Run-2, and enable it for all the Run-3 and Phase-2 workflows.

Reply:
Agree, thx. 
This can come later after this PR is merge.